kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-widgets/src/test/java/org/kie/workbench/common/stunner/client/widgets/menu/dev/MenuDevCommandBuilderTest.java

```diff
@@ -0,0 +1,43 @@
+package org.kie.workbench.common.stunner.client.widgets.menu.dev;
+
+import java.lang.annotation.Annotation;
+
+import com.google.gwt.dom.client.AnchorElement;
+import com.google.gwtmockito.GwtMockitoTestRunner;
+import com.google.gwtmockito.WithClassesToStub;
+import org.assertj.core.api.Assertions;
+import org.gwtbootstrap3.client.ui.AnchorListItem;
+import org.gwtbootstrap3.client.ui.html.Text;
+import org.jboss.errai.ioc.client.api.ManagedInstance;
+import org.jboss.errai.ioc.client.api.builtin.ManagedInstanceProvider;
+import org.jboss.errai.ioc.client.container.IOC;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+
+@RunWith(GwtMockitoTestRunner.class)
+@WithClassesToStub({Text.class, AnchorListItem.class, AnchorElement.class, IOC.class})
+public class MenuDevCommandBuilderTest {
+
+    @Mock
```

Comment:
Doesn't make sense to annotate this with `@Mock` if you're creating your own instance in the `setUp()`